src/main/java/roomescape/controller/TimeController.java

```diff
@@ -0,0 +1,57 @@
+package roomescape.controller;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.dto.request.CreateTimeRequest;
+import roomescape.dto.response.TimeResponse;
+import roomescape.service.TimeService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/times")
+public class TimeController {
+
+    private static final String HEADER_LOCATION = "Location";
+    private static final String LOCATION_DEFAULT_VALUE = "/times/";
+
+    private final TimeService timeService;
+
+    public TimeController(final TimeService timeService) {
+        this.timeService = timeService;
+    }
+
+    @PostMapping
+    public ResponseEntity<TimeResponse> createTime(@RequestBody CreateTimeRequest request) {
+        TimeResponse response = timeService.createTime(request);
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.add(HEADER_LOCATION, LOCATION_DEFAULT_VALUE + response.id());
+
+        return ResponseEntity.status(HttpStatus.CREATED)
+                .headers(headers)
+                .body(response);
+    }
+
+    @GetMapping
+    public ResponseEntity<List<TimeResponse>> getTimes() {
+        List<TimeResponse> responses = timeService.getTimes();
+        return ResponseEntity.status(HttpStatus.OK)
+                .body(responses);
+    }
```

Comment:
```suggestion
    @GetMapping
    public ResponseEntity<List<TimeResponse>> getTimes() {
        List<TimeResponse> responses = timeService.getTimes();
        return ResponseEntity.ok(responses);
    }
```
1. 아래에 있는 방법을 알고 계셨나요?
2. 둘 중에 어느 방법을 더 선호하시나요?

Reply:
1. 몰랐습니다.
2. status 언급없이 status 상태를 바로 체이닝으로 연결할 수 있군요! 더 간결해서 좋은 것 같습니다.

[질문 🙋🏻‍♀️]
delete 하는 메서드도 아래와 같이 변경해봤는데 어떤가요?

```java
   @DeleteMapping("/{timeId}")
    public ResponseEntity<Void> deleteTime(@PathVariable long timeId) {
        timeService.deleteTime(timeId);
        return ResponseEntity.noContent()
                .build();
    }
```

Reply:
해당 부분 언급하는 걸 누락했는데, 러키가 알아보고 개선했네요!
보기 좋습니다!